src/js/_enqueues/lib/gallery.js

```diff
@@ -88,7 +88,7 @@ jQuery( function($) {
 	}
 });
 
-jQuery(window).on( 'unload', function () { window.tinymce = window.tinyMCE = window.wpgallery = null; } ); // Cleanup.
+jQuery(window).on( 'pagehide', function () { window.tinymce = window.tinyMCE = window.wpgallery = null; } ); // Cleanup.
```

Comment:
I think we should look into removing this instance entirely. Otherwise, if a tab is restored then it would be broken. And it wouldn't be possible to restore these variables with `pageshow` since they are gone.

Reply:
This change was introduced in r9894

Reply:
Having a hard time finding where this file is even used in core.

Reply:
It is enqueued in two functions: `wp_media_upload_handler()` and `media_upload_gallery()`. This latter function refers to the screen as the "legacy media uploader form".

Reply:
OK, I found out how to get there:

0. Activate the Classic Editor plugin
1. Create a new post
2. Upload _new_ photos in the post and create a gallery from them
3. Save the post
4. With the post ID in hand, go to `/wp-admin/media-upload.php?type=gallery&tab=type&post_id={POST_ID}`

It looks like this:

![image](https://github.com/WordPress/wordpress-develop/assets/134745/ff8b9593-1ccd-4ce8-9630-e0d64cbc083a)

I don't see any other way to get to that screen.

Given this, either we:

1. Replace `unload` with `pagehide` and add a `pageshow` that does the reload behavior below, or
2. We just remove the line of cleanup code altogether. It seems like it was put in place for an iOS Safari bug from over a decade ago, similar to https://github.com/WordPress/wordpress-develop/pull/5056/files#r1332289674, so it's probably obsolete.

@azaozz Since you introduced this line of code and are probably the most familiar with this codebase, what do you think we should do (also considering this is an inaccessible legacy screen)?

Reply:
See also how @spenserhale removed similar code in `src/js/_enqueues/wp/mce-view.js`: https://github.com/WordPress/wordpress-develop/pull/5056/files#r1332287807

Reply:
I'm just going to go ahead and remove this for consistency with `src/js/_enqueues/wp/mce-view.js`.